controllers/argocd/notifications/rolebinding.go

```diff
@@ -0,0 +1,104 @@
+package notifications
+
+import (
+	"reflect"
+
+	"github.com/argoproj-labs/argocd-operator/pkg/argoutil"
+	"github.com/argoproj-labs/argocd-operator/pkg/cluster"
+	"github.com/argoproj-labs/argocd-operator/pkg/permissions"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func (nr *NotificationsReconciler) reconcileRoleBinding() error {
+
+	nr.Logger.Info("reconciling roleBindings")
+
+	name := argoutil.GenerateUniqueResourceName(nr.Instance.Name, nr.Instance.Namespace, ArgoCDNotificationsControllerComponent)
+	sa, err := permissions.GetServiceAccount(name, nr.Instance.Namespace, nr.Client)
+
+	if err != nil {
+		nr.Logger.Error(err, "reconsileRoleBinding: failed to get serviceaccount", "name", name, "namespace", nr.Instance.Namespace)
+		return err
+	}
+
+	roleBindingRequest := permissions.RoleBindingRequest{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        name,
+			Namespace:   nr.Instance.Namespace,
+			Labels:      nr.Instance.Labels,
```

Comment:
we should call util.DefaultLabels here to get all the labels we need

Reply:
Global value and only call `DefaultLabels` once in the reconcile() function in `notifications.go`.